TypeScript definition included dependency on redux-actions

Version: 3.0.0-beta.10

When compiling a TypeScript application that references this library - we receive the following error:
```
 ERROR in [at-loader] ./node_modules/redux-oidc/index.d.ts:3:40
          TS2307: Cannot find module 'redux-actions'.
```

This appears to be caused by the fact that the TypeScript definition file includes an import for 'redux-actions', yet 'redux-actions' is not listed as a dependency in `package.json`.

Two possible solutions would be to either add a dependency on 'redux-actions' to this library, or remove the references to it from the TypeScript definition file. We would be happy to submit a PR if you could indicate a preference.

--- a/index.d.ts
+++ b/index.d.ts
@@ -1,27 +1,35 @@
 declare module 'redux-oidc' {
     import { UserManager, UserManagerSettings, User } from 'oidc-client';
-    import { Action, BaseAction } from 'redux-actions';
     import { Map, fromJS } from 'immutable';
     import { Middleware, Store } from 'redux';
     import * as React from 'react';
 
+    export interface BaseAction {
+        readonly type: string;
+    }
+
+    export interface Action<Payload> extends BaseAction {
+        readonly payload?: Payload;
+        readonly error?: boolean;
+    }
+
     export interface UserState {
-        user?: User;
-        isLoadingUser: boolean;
+        readonly user?: User;
+        readonly isLoadingUser: boolean;
     }
 
     export interface CallbackComponentProps {
-        userManager: UserManager;
-        successCallback: (user: User) => void;
-        errorCallback?: (error: Error) => void;
-        route?: string;
+        readonly userManager: UserManager;
+        readonly successCallback: (user: User) => void;
+        readonly errorCallback?: (error: Error) => void;
+        readonly route?: string;
     }
 
     export class CallbackComponent extends React.Component<CallbackComponentProps> { }
 
     export interface OidcProviderProps<TSTate> {
-        userManager: UserManager;
-        store: Store<TSTate>;
+        readonly userManager: UserManager;
+        readonly store: Store<TSTate>;
     }
 
     export class OidcProvider<TState> extends React.Component<OidcProviderProps<TState>> { }

